docs/reference/esql/functions/examples/st_within.asciidoc

```diff
@@ -2,6 +2,8 @@
 
 *Example*
 
+
+
```

Comment:
This seems maybe bad.

Reply:
Like, not bad, but something that we shouldn't do on accident.

Reply:
++
we can avoid it